portfolio/src/main/webapp/comment-form.js

```diff
@@ -1,5 +1,25 @@
 const DEFAULT_COMMENT_COUNT = 3;
 
+/** Update the page upon page load */
+function init() {
+  checkLogin();
+  getComments(); 
+}
+
+/** Ensure the comment form is only displayed if the user is logged in */
+async function checkLogin() { 
+  const response = await fetch('/login-status');
+  const status = await response.json();
```

Comment:
This is not actually JSON, so as it is now, it should be `response.text()`

Reply:
changed to proper JSON